### ISSUE TITLE
Normalize depth of imports

### ISSUE THREAD

**Issue by erikrose** [erikrose/blessings#77]
On master, we deep-import most things so we can be briefer and more readable in the actual code: keep the noise up top. Only things that would be ambiguous if deep-imported are shallow-imported.

Blocks #72.

**Comment by jquast** [comment]
Can you clarify? Do you wish to see:
```
-from .formatters import (...)
+from blessings.formatters import (...)
```

**Comment by jquast** [comment]
ping

**Comment by erikrose** [comment]
Either of those is fine. I have a slight preference for the second, but it's probably just legacy.

**Comment by jquast** [comment]
This effort is included in branch ``blessed-integration-pep257-docstrings``, as re-ordering imports would cause merge conflicts or further branch-on-branching due to pending PR's.

**Comment by jquast** [comment]
Closed by #100, review at http://1984.ws/blessings/index.html
